charts/scalardb-cluster/templates/scalardb-cluster/certmanager.yaml

```diff
@@ -0,0 +1,83 @@
+{{- if .Values.scalardbCluster.tls.certManager.enabled }}
+{{- if not .Values.scalardbCluster.tls.certManager.issuerRef }}
```

Comment:
If you don't specify `Issuer` in the `issurRef` configuration, this chart deploys self-signed CA to generate certificates.

If you specify the `issuerRef` configuration, this chart doesn't deploy self-signed CA.